### DISCUSSION TITLE
Unable to set up Supabase locally

### DISCUSSION THREAD

**Discussion by Olaleye-Blessing** [supabase/supabase#15432] [Questions]
I got this error while setting up the repo locally.

node version: 18.16.1
npm version: 9.5.1
OS: Windows

![Screenshot (686)](https://github.com/supabase/supabase/assets/70102539/4a269438-00e9-4686-ab6c-925602b8497c)
![Screenshot (687)](https://github.com/supabase/supabase/assets/70102539/899b92e1-95cb-4632-ae6e-44c750b7e7ed)
![Screenshot (688)](https://github.com/supabase/supabase/assets/70102539/6810992b-1c69-491b-87e2-f7605fdb7eba)

**Comment by HarshitSinghT0mar** [comment]
I am getting error while running docs app, it says use ESm syntax in /node_modules/@supabase/auth-helpers-nextjs/dist/index.js , which i did, but it didn't work also i tried using use strict, that also didn't seem to work. What am I doing wrong. given www app is working fine 

<img width="959" alt="SupaBaseError" src="https://github.com/supabase/supabase/assets/119979451/882e3230-4920-4c03-983d-8868d9dba1f7">
